tests/containers/explorer/detail/helpers/explorerQuarters-test.js

```diff
@@ -60,6 +60,14 @@ describe('explorerQuarters', () => {
                 year: 1911
             });
         });
+        it('should use the previous fiscal year as the fiscal year on October 1st', () => {
```

Comment:
Can we generalize the description to `should use the previous fiscal year as the FY when the current date is between Oct 1 and Feb 14`? (You can still use Oct 1 as the test value). Similar for FY helper